### PR TITLE
[srvls][SRVKE-788] Add additional Kafka SASL docs

### PR DESCRIPTION
**Pull request by abrennan89**
Applies for 4.6+

New content preview link: https://deploy-preview-32689--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_workflows/serverless-kafka.html#serverless-kafka-sasl-public-certs_serverless-kafka

### CODE REVIEW

**Comment by abrennan89** [comment]
New content preview link: https://deploy-preview-32689--osdocs.netlify.app/openshift-enterprise/latest/serverless/event_workflows/serverless-kafka.html#serverless-kafka-sasl-public-certs_serverless-kafka

File: modules/serverless-kafka-sasl-public-certs.adoc [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+If you want to use SASL with public CA certificates, you must use the `tls.enabled=true` flag, rather than the `ca.crt` argument, when creating the secret. For example:
+
+[source,terminal]
+----
+$ oc create secret --namespace <namespace> generic <kafka_auth_secret> \
+    --from-literal=tls.enabled=true \
+  --from-literal=password="SecretPassword" \
+  --from-literal=saslType="SCRAM-SHA-512" \
+  --from-literal=user="my-sasl-user"
+----
```
Comment (matzew) [inline modules/serverless-kafka-sasl-public-certs.adoc:17]: The 
```
    --from-literal=tls.enabled=true \
```

looks like is having two spaces more than the other arguments

**Review by matzew — APPROVED** [review]
/lgtm
/approve

This PR does look good!

File: modules/serverless-kafka-sasl-public-certs.adoc (hunk 1 above)
Comment (abrennan89) [reply modules/serverless-kafka-sasl-public-certs.adoc]: Fixed 😄

**Comment by mgencur** [comment]
QE Approved

**Review by bergerhoffer — COMMENTED** [review]
LGTM!

**Pull request merged**